src/forms/css/forms.css

```diff
@@ -50,6 +56,11 @@
 .pure-form select:focus:invalid:focus {
   border-color: #e9322d;
 }
+.pure-form input[type="file"]:focus:invalid:focus,
```

Comment:
Pardon my 5 years late ping, but I am super curious about this selector and why `:focus:invalid` would not alone suffice? Cheers!